loader/src/main/scala/com/actian/spark_vector/loader/parsers/Parser.scala

```diff
@@ -394,7 +404,15 @@ object Parser extends scopt.OptionParser[UserOptions]("spark-submit --class com.
         if (cur.extractor(options).isEmpty) acc :+ cur.longName else acc
       })
     if (missing.isEmpty) {
-      success
+        val instance = "[a-zA-Z]+".r
```

Comment:
Why does the regular expression for instance exclude numerical digits?
If the instance is equivalent to the "installation code", then, according to [doc](https://docs.actian.com/vector/6.0/index.html#page/Connectivity%2FVector_Instance.htm%23ww229562)
the following holds:
"The installation code is an installer-defined, two-character code that identifies the installation, whether it be a server or client installation,
[...]
"The first character must be a letter; the second character can be a letter or numeral. If there is more than one installation on the same node, each installation must have a unique installation code."

so maybe something like
`val instance = "[a-zA-Z]([a-zA-Z]|[0-9])".r`
could work